components/CheckoutWizard/CancelButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "@equinor/eds-core-react"
+import { FunctionComponent } from "react"
+import { useIntl } from "react-intl"
+
+import { Link } from "../Link"
+
+import type { AssetIdProp } from "./types"
+
+export const CancelButton: FunctionComponent<AssetIdProp> = ({ assetId }) => {
+  const intl = useIntl()
+
+  return (
+    <Link href={{ pathname: "/assets/[id]", query: { id: assetId } }}>
```

Comment:
renders `<a><a></a></a>`. our current `Link` component doesn't support `passHref` to children that are valid components. have created an issue to deprecate the `Link` component (#189)

i suggest we just use the next.js `Link` component going forward, as we've discussed earlier.

Reply:
Djeeeez, right, I remember 😅  (Didn't test the markup 👀 )